build.gradle

```diff
@@ -39,6 +39,7 @@ dependencies {
     compile 'org.jenkins-ci.plugins:mailer:1.16@jar'
     compile 'org.jenkins-ci.plugins:mask-passwords:2.8@jar'
     compile 'com.amazonaws:aws-java-sdk:1.11.178'
+    compile 'com.splunk.splunkins:splunk-devops:1.6.4@jar'
```

Comment:
Do we have any automation yet for alerting us when new versions of these dependencies become available?  VersionEye has some support for tracking Gradle dependencies, although it's in the process of being shut down.  There are apparently also some [Gradle plugins](https://github.com/ben-manes/gradle-versions-plugin) for this.

Reply:
No, unfortunately, we don't. Jenkins itself keeps you up to date with plugin patches/etc. It's not perfect, but it's something.